packages/interactivity-router/src/index.js

```diff
@@ -1,7 +1,16 @@
+/**
```

Comment:
Nitpick: WordPress dependencies are always listed before internal dependencies by convention: https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md#imports

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/59707/commits/455b8d74e4b9fd2dabe3960b6d168a0791830546).